### ISSUE TITLE
Reproducing Overfitting

### ISSUE THREAD

**Issue by suptimq** [yuxumin/PoinTr#100]
Hi, I am trying to reproduce the results on the PCN dataset (i.e., Table 3 in the paper). While I was following the instructions for training GRNet, I got something similar to overfitting. I was wondering if this is normal

"""bash
CUDA_VISIBLE_DEVICES=0,1 bash ./scripts/dist_train.sh 2 13232 \
    --config ./cfgs/PCN_models/GRNet.yaml \
    --exp_name example
"""

![image](https://user-images.githubusercontent.com/29498118/226780489-57723057-b211-4aac-937a-a4bc09ec08d5.png)

**Comment by yuxumin** [comment]
Yes, the loss on trianing and evaluation phase is different. (BTW, why not try AdaPoinTr or PoinTr)

**Comment by suptimq** [comment]
Thanks for answering the question! That said, if the curve looks fine, what is the potential problem that makes it overfitting? 

What I was trying to do is to reproduce GRNet, PCN, and PoinTr results using the PCN dataset (i.e., Table 3 in the paper) but couldn't achieve a comparable performance for GRNet and PCN using the default config files. Is there anything I missed?

Yes, I did try AdaPoinTr and PoinTr and they were looking good.
